src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndHashWithDialogCodeAction.cs

```diff
@@ -95,21 +98,21 @@ protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperation
             public override string Title
                 => GenerateEqualsAndGetHashCodeAction.GetTitle(_generateEquals, _generateGetHashCode) + "...";
 
-            protected override async Task<Solution> GetChangedSolutionAsync(CancellationToken cancellationToken)
+            protected override async Task<Solution?> GetChangedSolutionAsync(CancellationToken cancellationToken)
```

Comment:
feels odd. can derived method not provide strong guarantees?

Reply:
Only on top-level nullability.